entries/s/sezzle.com.json

```
@@ -0,0 +1,13 @@
+{
+  "Sezzle": {
+    "domain": "sezzle.com",
+    "tfa": [
+      "sms"
+    ],
+    "notes": "Only merchants can enable 2FA.",
+    "documentation": "https://merchant-help.sezzle.com/hc/sections/16790628725780",
```

```suggestion
    "documentation": "https://merchant-help.sezzle.com/hc/en-us/articles/16960335317908",
```

This page seems to be better.